src/SignalR/clients/csharp/Client.SourceGenerator/src/gen/GeneratorHelpers.cs

```diff
@@ -27,5 +25,15 @@ public static string GetAccessibilityString(Accessibility accessibility)
                     return null;
             }
         }
+
+        public static string SourceFilePrefix()
```

Comment:
Curious: Why is this a `public` type nested in an `internal` one instead of a top-level `internal` class❔

Reply:
Oops, read this totally wrong. I need food&hellip;